Makefile

```diff
@@ -162,6 +162,9 @@ ifabric: $(IOS_PROJ_PATH)
 idocument:
 	OUTPUT=$(OUTPUT) ./platform/ios/scripts/document.sh
 
+style-code-ios:
+	node platform/ios/scripts/generate-style-code.js
```

Comment:
The script generates an API for both iOS and macOS. Let’s call it `style-code-darwin` and put the script in the platform/darwin/ directory.